scripts/_events.js

```diff
@@ -114,12 +115,15 @@ function listenForIbcChannelEvents(network, source, dispatcher) {
 
 function filterPacketEvents(portAddress, network) {
     const sendPacketConfig = config.sendPacket;
-    const sendUniversalPacketConfig = config.sendUniversalPacket;
-
-    const filterCondition = 
-        portAddress === sendPacketConfig[network].portAddr ||
-        portAddress === sendUniversalPacketConfig[network].portAddr;
-    return filterCondition;
+    let ucHandlerAddr;
```

Comment:
Recommend to use getUcHandler from `_get_vibc_src.js` to ensure standard patterns

Reply:
So getUcHandler fetches the contract itself, not just the address. But I do see your point and could maybe add a separate helper function getUcHandlerAddr instead?